automation/startjanssendemo.sh

```
@@ -38,8 +38,13 @@ fi
 sudo apt-get update
 sudo snap install microk8s --classic
 sudo microk8s.status --wait-ready
-sudo microk8s.enable dns registry ingress hostpath-storage helm3
-sudo microk8s kubectl get daemonset.apps/nginx-ingress-microk8s-controller -n ingress -o yaml | sed -s "s@ingress-class=public@ingress-class=nginx@g" | microk8s kubectl apply -f -
+sudo microk8s.enable dns registry hostpath-storage helm3
+sudo microk8s enable metallb:$EXT_IP/32
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)